### ISSUE TITLE
[table] Accessibility: Table headers cannot be accessed/triggered from keyboard

### ISSUE THREAD

**Issue by andresjoelv** [tyler-technologies-oss/forge#63]
**Describe the feature:**
When using the keyboard to access the table, header and its controls are not reachable. This makes screenreaders and keyboard navigation skip to the content of the table. It would be nice if we can access this controls from keyboard input.
Also aria labels

**Additional context:**
The table currently handles mouse interactions (it uses mousedown and mouseup events to trigger this).
<img width="927" alt="Screen Shot 2022-05-31 at 8 35 36 PM" src="https://user-images.githubusercontent.com/11197984/171422263-e9ab63de-0fc2-4ff7-960c-8fb340f953ea.png">
